core/src/main/java/org/kiwix/kiwixmobile/core/utils/SharedPreferenceUtil.kt

```diff
@@ -138,13 +139,14 @@ class SharedPreferenceUtil @Inject constructor(context: Context?) {
   var hostedBooks: Set<String>?
     get() = sharedPreferences.getStringSet(PREF_HOSTED_BOOKS, HashSet())
     set(hostedBooks) {
-      sharedPreferences.edit().putStringSet(PREF_HOSTED_BOOKS, hostedBooks).apply()
+      sharedPreferences.edit { putStringSet(PREF_HOSTED_BOOKS, hostedBooks) }
     }
 
   var textZoom: Int
     get() = sharedPreferences.getInt(TEXT_ZOOM, DEFAULT_ZOOM)
     set(textZoom) {
-      sharedPreferences.edit().putInt(TEXT_ZOOM, textZoom).apply(); textZooms.offer(textZoom)
+      sharedPreferences.edit { putInt(TEXT_ZOOM, textZoom) }
+      textZooms.offer(textZoom)
     }
```

Comment:
Here as well

Reply:
you couldn't do it here without rewriting this to be 1 line